openedx/adg/admin_dashboard/admin_settings/views.py

```diff
@@ -228,6 +254,50 @@ def get_queryset(self):
 
         return admin_users
 
+    def get_status(self, action_type, email):
+        """
+        Return status based on the action the user performed and in case of invitation update return an error if no
+        admin is found against the email.
+
+        Arguments:
+            action_type (str): indicates the type of action the user performed
+            email (str): email of the admin being invited/updated/removed
+
+        Returns:
+            int: invitation status
+            Invitation: Invitation object
+            str: error
+        """
+        if action_type == INVITATION:
+            return Invitation.PENDING, None, ''
+        elif action_type == EDIT_INVITATION:
+            user = User.objects.filter(email=email).first()
+
+            if not user:
+                return '', None, NO_ADMIN_ERROR.format(email=email)
```

Comment:
Doc string says it will return int but here its returning empty string. Is it intentional?